pywr/parameters/_hydropower.pxd

```diff
@@ -0,0 +1,16 @@
+from .._core cimport Timestep, Scenario, ScenarioIndex, AbstractNode, Storage, AbstractStorage
+from ._parameters cimport Parameter, IndexParameter
+
+
+
+cdef class HydroPowerTargetParameter(Parameter):
```

Comment:
HydroPower vs Hydropower?

Reply:
Agreed should be the later. For consistency we will need to change the parameters that are merged already too.